stdlib/LinearAlgebra/src/tridiag.jl

```diff
@@ -608,6 +612,10 @@ transpose(S::Tridiagonal{<:Number}) = Tridiagonal(S.du, S.d, S.dl)
 Base.copy(aS::Adjoint{<:Any,<:Tridiagonal}) = (S = aS.parent; Tridiagonal(map(x -> copy.(adjoint.(x)), (S.du, S.d, S.dl))...))
 Base.copy(tS::Transpose{<:Any,<:Tridiagonal}) = (S = tS.parent; Tridiagonal(map(x -> copy.(transpose.(x)), (S.du, S.d, S.dl))...))
 
+ishermitian(S::Tridiagonal{<:Real}) = S.du == S.dl
+ishermitian(S::Tridiagonal{<:Complex}) = all(x -> imag(x) == 0, S.d) && S.du == adjoint(S.dl)[1,:]
```

Comment:
```suggestion
ishermitian(S::Tridiagonal) = S.du == adjoint.(S.dl)
```

Reply:
I think this is missing `&& isreal(S.d)`.